plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearch6ConnectorTest.java

```diff
@@ -47,6 +49,12 @@ public void testIndexWithMappingsButNoProperties()
         assertTableDoesNotExist(indexName);
     }
 
+    @Test
+    public void testNoAggregationPushdown()
+    {
+        assertThat(query("SELECT count(*) FROM nation")).isNotFullyPushedDown(AggregationNode.class);
```

Comment:
Can we use another aggregate here?  It just so happens that `count(*)` is actually pushed down (though not fully), so I think using something else may avoid potential confusion.  See `CountQueryPageSource`.